### ISSUE TITLE
Coverage of common concepts and entities

### ISSUE THREAD

**Issue by hankcs** [yago-naga/yago4#19]
Dear authors,

I've read the yago4 paper which looks very interesting to me. I have a question regarding its coverage of some common items. When I query "rain" and "iPad" on yago website neither of them can be found. Is that normal? I'm quite new to the KB field but I'd expect them to be covered as they occur so frequently in the real world data.

**Comment by fmsuchanek** [comment]
Yes, we cover only named entities, so it's normal that "rain" is out :-) For the iPad, we're currently preparing a new version of YAGO that has it :-)
